conformance_pack/backup.sp

```diff
@@ -186,3 +206,63 @@ query "backup_recovery_point_min_retention_35_days" {
       aws_backup_recovery_point;
   EOQ
 }
+
+query "backup_plan_region_configured" {
+  sql = <<-EOQ
+    with count_plans as (
+      select
+        region,
+        account_id,
+        count(*) as count
+      from
+        aws_backup_plan
+      group by
+        region,
+        account_id
+    )  
+    select
+      'arn:' || r.partition || '::' || r.region || ':' || r.account_id as resource,
+      case
+        when cp.count > 0 then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when cp.count > 0 then cp.count || ' backup plan(s) exist in region ' || r.region || '.'
+        else 'No backup plans exist in region ' || r.region || '.'
+      end as reason
```

Comment:
Do we need to add the region information? Won't it be already available in the common dimension?